package.json

```diff
@@ -3,18 +3,21 @@
   "license": "AGPL-3.0-or-later",
   "private": true,
   "devDependencies": {
-    "@trivago/prettier-plugin-sort-imports": "^4.2.0",
+    "@trivago/prettier-plugin-sort-imports": "^4.1.1",
     "@typescript-eslint/eslint-plugin": "^5.59.8",
     "@typescript-eslint/parser": "^5.44.0",
-    "@types/node": "^18.7.15",
-    "eslint": "^8.51.0",
+    "eslint": "^8.41.0",
     "eslint-config-prettier": "^8.8.0",
-    "prettier": "^2.4.1",
-    "prettier-plugin-solidity": "^1.1.1",
-    "typescript": "^4.9.5"
+    "prettier": "^2.8.8",
+    "prettier-plugin-solidity": "^1.1.3",
+    "solhint": "^3.6.2",
+    "solhint-plugin-prettier": "^0.0.5",
+    "typescript": "5.0.4"
```

Comment:
Note, that this isn't necessary [here](https://github.com/aragon/osx-plugin-template-hardhat/blob/6b3bb1c26aee99822fedccde402c4750e773d1e0/packages/contracts/package.json#L39) and I don't understand why